docs/sources/setup/install/helm/deployment-guides/azure.md

```diff
@@ -0,0 +1,588 @@
+- Azure CLI installed on your local machine. Refer to [Installing the Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli). This is a requirement for following this guide as all resources will be created using the Azure CLI.
+  
+### AKS Minimum Requirements
+
+Before creating an AKS cluster in Azure you need to create a resource group. You can create a resource group using the Azure CLI:
+
+```bash
+az group create --name <INSERT-NAME> -location <INSERT-AZURE-REGION>
+```
+
+{{< admonition type="caution" >}}
+These AKS requirements are the minimum specification needed to deploy Loki using this guide. You may wish to adjust the instance types based on your Azure environment and workload. **If you choose to do so, we cannot guarantee that this sample configuration will still meet your needs.**
+
+In this guide, we deploy Loki using `Standard_E2ds_v5` instances. This is to make sure we remain within the free tier limits for Azure. Which allows us to deploy up to 10 vCPUs within a region. We recommend for large production workloads to scale these nodes up to `Standard_D4_v5`.
+{{< /admonition >}}
+
+The minimum requirements for deploying Loki on AKS are: 
+
+- Kubernetes version `1.30` or above.
+- `3` nodes for the AKS cluster.
+- Instance type depends on your workload. A good starting point for a production cluster in the free tier is `Standard_E2ds_v5` instances and for large production workloads `Standard_D4_v5` instances.
+
+Here is how to create an AKS cluster with the minimum requirements:
+
+```bash
+az aks create \
+  --resource-group <MY_RESOURCE_GROUP_NAME> \
+  --name <MY_AKS_CLUSTER_NAME> \
+  --node-count 3 \
+  --node-vm-size Standard_E2ds_v5 \
+  --generate-ssh-keys \
+  --enable-workload-identity \
+  --enable-oidc-issuer
+```
+
+Note in the above command we have enabled workload identity and OIDC issuer. This is required for the Loki service account to authenticate with Azure AD. If you have already created an AKS cluster, you can enable these features using the following command:
+
+```bash
+az aks update \
+  --resource-group <MY_RESOURCE_GROUP_NAME> \
+  --name <MY_AKS_CLUSTER_NAME> \
+  --enable-workload-identity \
+  --enable-oidc-issuer
+```
+
+The Azure CLI also allows you to bind the AKS cluster to kubectl. You can do this by running the following command:
+
+```bash
+az aks get-credentials --resource-group <MY_RESOURCE_GROUP_NAME> --name <MY_AKS_CLUSTER_NAME>
+```
+
+## Configuring Azure Blob Storage
+
+{{< admonition type="tip" >}}
+ **Consider** using unique bucket names rather than;  `chunk`, `ruler` and `admin`. Although Azure Blog Storage is not directly affected by this [security update](https://grafana.com/blog/2024/06/27/grafana-security-update-grafana-loki-and-unintended-data-write-attempts-to-amazon-s3-buckets/) it is best practise to use unique container names (buckets).
```

Comment:
```suggestion
 Consider using unique bucket names rather than:  `chunk`, `ruler` and `admin`. Although Azure Blog Storage is not directly affected by this [security update](https://grafana.com/blog/2024/06/27/grafana-security-update-grafana-loki-and-unintended-data-write-attempts-to-amazon-s3-buckets/) it is a best practice to use unique container names for buckets.
```